MedicationScheduler/AuthApp/views.py

```diff
@@ -61,3 +60,27 @@ def login_view(request):
 def logout_view(request):
     logout(request)
     return redirect('Auth:login')
+
+def profile_view(request):
+    user_update_form = UpdateUserForm(instance=request.user)
+    password_change_form = PasswordChangingForm(request.user)
+    update_profile_form = UpdateUserProfile()
+
+    if request.method == 'POST' and 'change_password' in request.POST:
+        password_change_form = PasswordChangingForm(request.user, request.POST)
+        if password_change_form.is_valid():
+            password_change_form.save()
+            return redirect('Auth:login')
+    elif request.method == 'POST' and 'user_update' in request.POST:
+        user_update_form = UpdateUserForm(request.POST, instance=request.user)
+        if user_update_form.is_valid():
+            user_update_form.save()
+    elif request.method == 'POST' and 'update_profile_info' in request.POST:
+        update_profile_form = UpdateUserProfile(request.POST, instance=request.user)
+        if update_profile_form.is_valid():
+            update_profile_form.save()
```

Comment:
User profile information is not being saved into the database on my end.

Reply:
@bew5294 good catch. thank you! It should work now